src/@types/html.d.ts

```diff
@@ -0,0 +1,5 @@
+// Declaring a module with *.html due to us
+// not having a default import w/ html
+// if we define properties / export a default it'll
+// compile to `require('thing.html').default` lol
+declare module '*.html';
```

Comment:
I'm confused... how come this is needed? o.O

Reply:
That is not in the latest commit.